autoload/SpaceVim/layers/lang/go.vim

```diff
@@ -0,0 +1,12 @@
+function! SpaceVim#layers#lang#go#plugins() abort
+    if has('nvim')
+        return ['zchee/deoplete-go', {'on_ft' : 'go', 'build': 'make'}]
+    else
+        return ['fatih/vim-go', { 'on_ft' : 'go', 'loadconf_before' : 1}]
```

Comment:
Why not load `vim-go` no matter what? As far as I know it's neovim compatible (support for neovim is in beta but still) and it has more than just completion but running `go fmt` on save or `go vet`/`go imports`.